DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py

```diff
@@ -532,48 +532,53 @@
 #--------------------------------------------------
 
 print("Running with run type = ", process.runType.getRunType())
+
 ### HEAVY ION SETTING
-if (process.runType.getRunType() == process.runType.hi_run):
-    process.castorDigis.InputLabel = "rawDataRepacker"
-    process.csctfDigis.producer = "rawDataRepacker"
-    process.dttfDigis.DTTF_FED_Source = "rawDataRepacker"
-    process.ecalDigis.cpu.InputLabel = "rawDataRepacker"
-    process.ecalPreshowerDigis.sourceTag = "rawDataRepacker"
-    process.gctDigis.inputLabel = "rawDataRepacker"
-    process.hcalDigis.InputLabel = "rawDataRepacker"
-    process.muonCSCDigis.InputObjects = "rawDataRepacker"
-    process.muonDTDigis.inputLabel = "rawDataRepacker"
-    process.muonRPCDigis.InputLabel = "rawDataRepacker"
-    process.scalersRawToDigi.scalersInputTag = "rawDataRepacker"
-    process.siPixelDigis.cpu.InputLabel = "rawDataRepacker"
-    process.siStripDigis.ProductLabel = "rawDataRepacker"
-    process.siStripFEDMonitor.RawDataTag = "rawDataRepacker"
+if process.runType.getRunType() == process.runType.hi_run:
+    rawDataRepackerLabel = 'rawDataRepacker'
```

Comment:
out of curiosity, why is this better?
Should we do the same for `rawDataCollector` in the pp section?

Reply:
I think it's subjective.. no strong opinion. I thought that using a variable..
 - allows to change only in one place if this ever needs to be changed
 - avoids the possibility of having a typo in one of N strings that were there before

Reply:
In 170b35b63fa3d84212175de73ace7c964fddc0dc, to be consistent, I also did the same for `rawDataCollector`. (if this is not really useful, I can undo it)